src/App.tsx

```diff
@@ -55,6 +57,34 @@ loader.init().then((monaco) => {
   })
 })
 
+const getAppDataFromSearchParams = () => {
```

Comment:
Add return type to function:
`const getAppDataFromSearchParams = (): AppData => {`